### PR TITLE
Adds Notifications section to README

### PR DESCRIPTION
**Pull request by ctorgalson**
- Also adds Update Status item to basic list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,8 @@ Below is a site launch checklist, with details on individual areas of interest t
 * Is Google Analytics (or your analytics tool of choice) configured?
 * Is robots.txt configured for production (ie. did you remove any changes that were made for development)?
 * Is Drupal's [internal page cache](https://www.drupal.org/documentation/modules/internal_page_cache) enabled?
-* Is the [Security Review](https://www.drupal.org/project/security_review) module installed and providing a clean report? 
+* Is Drupal's Update Status module enabled and configured?
+* Is the [Security Review](https://www.drupal.org/project/security_review) module installed and providing a clean report?
```
Comment (chrisfree) [inline README.md:20]: Looks like there isn't an official release for this module on D8?

File: README.md [hunk 2]
```diff
@@ -39,6 +40,10 @@ At a minimum, the following jobs should be configured in [Jenkins](https://jenki
 * Cron to be run at least once every 24 hours.
 * A Drush cache clear job that can be run on-demand from the Jenkins UI.
 
+### Notifications {#notifications}
+* Drupal sites should have the Update Manager module configured to send weekly emails about available security updates to a Slack channel (preferably one that the client belongs to).
+* Sites using HTTPS that *do not use Letsencrypt* should have a Jenkins job configured send weekly messages about upcoming certificate renewal dates to a Slack channel (preferably one that the client belongs to).
```
Comment (chrisfree) [inline README.md]: Should this be 
> do not use _Let's Encrypt_ should have a Jenkins job _to_ configured

**Review by chrisfree — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ctorgalson) [reply README.md:20]: It's been shipping in core since D6. We just have to turn it on.

File: README.md (hunk 2 above)
Comment (ctorgalson) [reply README.md]: Fixed by 87bed6a.

File: README.md (hunk 1 above)
Comment (chrisfree) [reply README.md:20]: @ctorgalson sorry, I'm talking about the `security_review` module. I realize now that you aren't changing that line at all.

**Review by chrisfree — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ctorgalson) [reply README.md:20]: Ah! I Get it now.

**Pull request merged**